elementary/clients/slack/schema.py

```diff
@@ -10,3 +14,12 @@ class SlackMessageSchema(BaseModel):
     text: Optional[str] = None
     attachments: Optional[list] = None
     blocks: Optional[list] = None
+
+    @validator("attachments", pre=True)
+    def validate_attachments(cls, attachments):
+        if isinstance(attachments, list) and len(attachments) > 50:
+            logger.error(
+                f"Slack message attachments limit is 50. You have {len(attachments)} attachments. Attachments were removed fro the message"
+            )
+            return None
```

Comment:
If we have 50+ attachments, instead of sending them, we send _nothing_ ? Why not at least send the first 50?

Reply:
I think that there is no reason that we will get to the point where the schema is the one that is removing the attachments.
This is more for handling bad usage while developing.
Login an error and removing the attachments is more noticeable and will make us attend the issue + I think there is no use of sending partial data that you can't control where the cut is (it can send weird messages).

As you can see, when sending the summary message, I handle this case and send different message when we pass the limit.

If you think that this is still an odd behaviour, please let me know and we can discuss it more 🙂

Reply:
I see- it's more defensive programing than a user facing behaviour
Makes sense